native/clesperantoj/src/kernelj.cpp

```diff
@@ -1,8 +1,14 @@
 #include "kernelj.hpp"
 
 #include "cleAddImageAndScalarKernel.hpp"
+#include "cleGaussianBlurKernel.hpp"
```

Comment:
In future we should replace with `cleKernelList.hpp` which include all kernel behind. This way we will not have to bother with this.

But for a first PR and how unstable the repo is, I suggest we keep to the classic approach for now